src/cmean_gaussian.jl

```diff
@@ -65,7 +65,7 @@ mean_var(p::CMeanGaussian, z::AbstractArray) = (mean(p, z), variance(p, z))
 function Flux.functor(p::CMeanGaussian{V,S,M}) where {V,S,M}
     fs = fieldnames(typeof(p))
     nt = (; (name=>getfield(p, name) for name in fs)...)
-    nt, y -> CMeanGaussian{V,S,M}(y...)
+    nt, y -> CMeanGaussian{V,S,typeof(gpu(p.mapping))}(y...)
```

Comment:
This only fixes the mapping conversion, however the `σ` field is still not converted.

Reply:
Also, this is probably going to break if one has `CuArrays` loaded but the model is actually on cpu.